pkg/executor/service.go

```diff
@@ -172,3 +203,16 @@ func serviceName(
 
 	return "svc-" + baseName
 }
+
+// serviceType is a utility function that converts serviceType string to
+// v1.ServiceType
+func serviceType(
```

Comment:
I think we could get rid of this logic now that the mutator makes sure that there's a value for this property.

Reply:
Oh wait, I see it's doing type conversion. Never mind.